src/gui/src/inspector.cpp

```diff
@@ -964,13 +968,63 @@ void Inspector::doubleClicked(const QModelIndex& index)
   emit indexDoubleClicked(index);
 }
 
+bool invokeEditorFromEnv(const FileLine& file_line)
+{
+  QString editorCommand
+      = QProcessEnvironment::systemEnvironment().value("EDITOR");
+
+  if (editorCommand.isEmpty()) {
+    qWarning() << "EDITOR environment variable not set.";
+    return false;
+  }
+
+  // Check if the placeholders are present in the command string.
+  if (!editorCommand.contains("%FILE%") || !editorCommand.contains("%LINE%")) {
```

Comment:
Since editor is a env var used in Linux, it seems like a bad idea to overload it with these substitutions. It doesn't seem like it would require much to add a few default programs emacs, gedit, etc based on the editor setting and then if a user wants to pick something else they can provide a Openroadeditor env var

Reply:
I agree, this was quick and dirty.  Maybe OPENROAD_EDITOR?  Alternatively we could have an command to configure it.

Reply:
I think `OPENROAD_EDITOR` is fine, it looks like we could inspect `EDITOR` and if it's set to a known program then we can load that.
`gedit +%LINE% %FILE%`
`emacs +%LINE% %FILE%`